ote/src/clj/ote/services/login.clj

```diff
@@ -27,16 +28,22 @@
 
 (defqueries "ote/services/login.sql")
 
-(defn- with-auth-tkt [response auth-tkt-value domain]
-  (update response :headers
-          assoc "Set-Cookie" (if (nil? domain)
-                               [(str "auth_tkt=" auth-tkt-value "; Path=/; HttpOnly")]
-                               ;; Three cookies are required to match ckan cookie configuration
-                               [(str "auth_tkt=" auth-tkt-value "; Path=/; HttpOnly; Secure")
-                                (str "auth_tkt=" auth-tkt-value "; Path=/; HttpOnly"
-                                     "; Domain=" domain "; Secure")
-                                (str "auth_tkt=" auth-tkt-value "; Path=/; HttpOnly"
-                                     "; Domain=." domain "; Secure")])))
+(defn ^:private unix-epoch []
+  ;; Returns a zoned date-time that can later be
+  ;; formatted as: "Thu, 1 Jan 1970 00:00:00 GMT"
+  (java-time.zone/zoned-date-time 1970 "UTC"))
+
+(defn- with-auth-tkt [response auth-tkt-value domain expiry-timestamp is-deleted]
+  (let [timestamp-string (java-time.format/format :rfc-1123-date-time (if is-deleted (unix-epoch) expiry-timestamp))]
```

Comment:
`unix-epoch` is a value now so don't call it.

Reply:
Fixed :)